CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+
+### Changed
+- Update the default behavior of NScale video uplink bandwidth policy to scale down resolution based on the number 
+  of videos.
```

Comment:
You might want more discussion about whether this is considered a breaking change, and if we don't consider it one, how to go back to the old behavior.

Reply:
Updated